cf-serverd/server.h

```diff
@@ -93,6 +92,22 @@ typedef struct
 
 } ServerAccess;
 
+#if 0
+struct peer_id
+{
+    char ipaddr[CF_MAX_IP_LEN];
+    char revdns[MAXHOSTNAMELEN];
+    Key *key;
+    /* The following are strings as provided by the peer. */
+    char *peer_username;
+    char *peer_hostname;
+    char *peer_ipaddr;
+    /* Any other identification string the peer might send */
+    char *peer_custom;
+};
+typedef struct peer_id Peer_ID;
+#endif
+
```

Comment:
I removed a bunch of other code from my unfinished refactorings, but left this one specifically to remember, because I consider it an important TODO. Hmmm maybe I'll move it to my personal notepad...